src/util/units.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#pragma once
+
+#define CLOCKS_PER_MILLISEC (CLOCKS_PER_SEC / 1000)
+#define MEMORY_HUMAN(x) ((x) / (double)(1024 * 1024))
```

Comment:
Can we drop it? or rename to `MEMORY_MB`?

Reply:
No, it's in use.
I'll rename